### ISSUE TITLE
Name server freezes after creating more than 45 agents

### ISSUE THREAD

**Issue by Karolina-Bogacka** [opensistemas-hub/osbrain#364]
I was just wondering why exactly that is and what should I do if I want to use more in my simulation

**Comment by Peque** [comment]
@Karolina-Bogacka It shouldn't be the case. :blush: 

Could you share a [Minimal, Reproducible Example](https://stackoverflow.com/help/minimal-reproducible-example)?

**Comment by ryanstwrt** [comment]
Also, you may want to see how many threads are currently allowed in Pyro4. I know that I needed to set:
`Pyro4.config.THREADPOOL_SIZE=<some_number>` to allow for a large number of agents.

**Comment by Peque** [comment]
@ryanstwrt Good point. By default it is set to 16 by osBrain, which may be insufficient for some applications.
